packages/Algernon.package/ALGCore.class/instance/handleListenEvent..st

```diff
@@ -1,7 +1,13 @@
 events-processing
 handleListenEvent: anEvent
-	| isActivationString shouldToggle |
+	| isActivationString shouldToggle timestamp difference |
+	timestamp := TimeStamp current time asNanoSeconds.
+	difference := timestamp - (lastHalfActivationTimestamp ifNil: [0]).
 	isActivationString := (anEvent keyString = self class activationString).
 	shouldToggle := (isActivationString and: [self receivedActivation not]).
-	shouldToggle ifTrue: [self toggleVisibility: anEvent. self receivedActivation: true].
+	shouldToggle ifTrue: [
+		(self class activationStringDoubleclick not or: [0 <= difference and: [difference <= self class maxHalfClickDeviation]]) ifTrue: [
```

Comment:
`difference between: 0 and: self class maxHalfClickDeviation`

Reply:
Should be also done by the refactoring 👍